lock.rst

```diff
@@ -269,12 +273,16 @@ provides :ref:`named lock <reference-lock-resources-name>`:
     .. code-block:: php
 
         // config/packages/lock.php
-        $container->loadFromExtension('framework', [
-            'lock' => [
-                'invoice' => ['semaphore', 'redis://r2.docker'],
-                'report' => 'semaphore',
-            ],
-        ]);
+        use Symfony\Config\FrameworkConfig;
+
+        return static function (FrameworkConfig $framework) {
+            $framework->lock()
+                ->resource('lock', [
+                    'invoice' => ['semaphore', 'redis://r2.docker'],
```

Comment:
We cannot use an array here... Hm..

> Invalid type for path "framework.lock.resources.lock.invoice". Expected "scalar", but got "array".

Reply:
@jderusse Can you look at this comment specifically?

Reply:
I think this should be
```
$framework->lock()
                ->resource('invoice', ['semaphore', 'redis://r2.docker'])
                ->resource('cron', 'flock')
```

Reply:
Oh, yes. I misunderstood the original config. Thank you

Reply:
beware, maybe the second case is not allowed by the PHP contract? because it should be an array

Reply:
Yes. You are correct. I updated the PR to use an array.